tfc operator setup updates (#587)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/kubernetes/setup.mdx

## Install and configure

> 1. Generate an [user or team](/terraform/cloud-docs/users-teams-organizations/api-tokens#access-levels) API token with permissions to manage workspaces, plan and apply runs, and upload configurations within Terraform Cloud and save it to a file. These instructions assume you are using a file named `credentials`.

Reviewer: ```suggestion
1. Generate a [user](/terraform/cloud-docs/users-teams-organizations/api-tokens#user-api-tokens) or [team](/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens) API token in Terraform Cloud. The user or team must have permission to create workspaces and apply runs. Save the token to a file named `credentials`.
```

Reviewer: @vravind1 I just wanted to double check these list of permissions. IIRC the operator can also create projects now, correct? So would the token instead need the org permissions of "Manage all workspaces" and "Manage all projects"?
